batterymonitor@pdcurtis/files/batterymonitor@pdcurtis/po/batterymonitor@pdcurtis.pot

```diff
@@ -326,6 +326,18 @@ msgid ""
 "  Icon Only: The battery icon on a colored background indicating the current status"
 msgstr ""
 
+#. 5.4->settings-schema.json->iconTheme->options
+msgid "Classic"
+msgstr ""
+
+#. 5.4->settings-schema.json->iconTheme->description
+msgid "Icon Theme"
+msgstr ""
+
+#. 5.4->settings-schema.json->iconTheme->tooltip
+msgid "Icon Theme"
+msgstr ""
+
```

Comment:
Did you regenerate these after you finished making your changes? It doesn't look like it's picked up all the modified strings correctly.

Reply:
ah I edited them manually

Reply:
You can use the scripts in the home directory to help you with these changes:
* `$ cinnamon-spices-makepot batterymonitor@pdcurtis` will generate the updated translations
* `$ validate-spice batterymonitor@pdcurtis` will run the same validation locally that will be run on the pull request so you can fix issues before they get to GitHub